bundles/multitenant-managed-api-service/1.22.0/bundle.Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM scratch
```

Comment:
no need for this file to be in.

Reply:
Is this file also unnecessary for standard versions of RHOAM? Right now, `scripts/prepare-release.sh` copies `bundle.Dockfile` into the `bundles/${OLM_TYPE}/${VERSION}/` directory but if neither version of RHOAM needs it, I'll get rid of the `cp` command from the prepare-release.sh script